libraries/radpdfprocessing/features/flatten-form-fields.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Flatten Form Fields
+page_title: Flatten Form Fields
+description: This article shows how one can flatten the form fields of a pdf document. 
+slug: radpdfprocessing-flatten-form-fields
```

Comment:
This functionality is related to the Interactive Forms and I think the topic should be in that section instead of inside Features

Reply:
This is a feature that is not part of the actual model